CMakeLists.txt

```diff
@@ -154,6 +154,9 @@ if(WW3)
   if(${CMAKE_Platform} STREQUAL "hera.intel")
     set(WW3_COMP "hera")
   endif()
+  if(${CMAKE_Platform} STREQUAL "orion.intel")
```

Comment:
We should really try to get rid of these extra machine dependencies when moving to cmake.